src/plugins/intel_gna/src/gna_plugin.cpp

```diff
@@ -852,10 +853,6 @@ void GNAPlugin::LoadNetwork(const CNNNetwork& _network) {
         }
         portId++;
     }
-    // TODO: how active list will work in multioutput case
-    // make room for active list
-    gnamem->getQueue(REGION_OUTPUTS)
```

Comment:
Was it not needed?

Reply:
When determining the required memory size or allocating memory, a check is performed which excludes this function call from being effective if the argument "_ptr_out" is a null pointer.

    /**
     * @brief allocates memory and updates pointers
     */
    void allocateRegion(GNAMemRequestsQueue* mRequests) {
        size_t r_size = ALIGN(mRequests->getSize(), _page_alignment);
        for (auto& re : mRequests->_mem_requests) {
            // skipping Bind, crossregion and empty requests
            if (re._type == REQUEST_BIND || re._ptr_out == nullptr)
                continue;